VideoIndexerEnabledByArc/aks/vi-edge-deployment-script.sh

```diff
@@ -179,78 +179,81 @@ fi
 
-      echo -e "\t create aks cluster Name: $aks , Resource Group $rg- ***done***"
-      echo "Adding another worload node pool"
-      az aks nodepool add -g $rg --cluster-name $aks  -n workload \
-              --os-sku AzureLinux \
-              --mode User \
-              --node-vm-size $workerVmSize \
-              --node-osdisk-size 100 \
-              --node-count 0 \
-              --max-count 10 \
-              --min-count 0  \
-              --tags $tags \
-              --enable-cluster-autoscaler \
-              --max-pods 110
-      echo "Adding another workload node pool ***done***"
-      #=============================================#
-      #============== AKS Credentials ==============#
-      #=============================================#
-      echo -e  "\tConnecting to AKS and getting credentials  - ***start***"
-      az aks get-credentials --resource-group $rg --name $aks --admin --overwrite-existing
-      echo "AKS connectivity Sanity test"
-      kubectl get nodes
-      echo -e "\tconnect aks cluster - ***done***"
-      #=============================================#
-      #============== Add ingress controller =======#
-      #=============================================#
-      echo -e "\tAdding ingress controller -- ***start***"
-      kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.9.4/deploy/static/provider/cloud/deploy.yaml
-      echo -e "\tAdding ingress controller -- ***done***"
-      #=============================================#
-      #========= Patch Public IP DNS Label =========
-      #=============================================#    
-      publicIpResourceId=$(az network public-ip list --resource-group $nodePoolRg --query "[?contains(name, 'kubernetes')].id" -otsv)
-      echo "publicIpResourceId: $publicIpResourceId"
-      az network public-ip update --ids $publicIpResourceId --dns-name ${resourcesPrefix}
-      echo "Public IP DNS Label has been updated to ${resourcesPrefix}"
-      
-      #=============================================#
-      #============== Create AKS ARC Cluster =======#
-      #=============================================#
-      echo -e "\tConnecting AKS to ARC-AKS -- ***start***"
-      az connectedk8s connect --name ${connectedClusterName} --resource-group $rg --yes
-      echo "Performing AKS-Arc-connected connectivity Sanity test"
-      connectedResult=$(az connectedk8s show --name ${connectedClusterName} --resource-group $rg 2>&1)
-      if [[ "$connectedResult" == *"ERROR"* ]]; then
-        echo "AKS-Arc-connected connectivity Sanity test failed. Exiting"
-        exit 1
-      fi
-      echo -e "\tconnecting AKS to ARC-AKS -- ***done***"
+  #=============================================#
+  #======== Create Connected-ARC Cluster =======#
+  #=============================================#
+  echo -e "\tConnecting AKS to ARC-AKS -- ***start***"
+  az group create -g ${rg} --location ${region} --tags $tags
```

Comment:
Isn't `az create group` redundant here? The aks cluster was created in this RG before